gui/screens/settings_messaging.py

```diff
@@ -56,15 +58,30 @@ def init_ui(self):
             elif str(getattr(child, 'objectName', '')) == 'chat-icon':
                 self._chat_button = Button(name='', real_name=driver.objectMap.realName(child))
 
+    @allure.step('Accept request')
     def accept(self) -> MessagesScreen:
         assert self._accept_button is not None, 'Button not found'
         self._accept_button.click()
         return MessagesScreen().wait_until_appears()
 
+    @allure.step('Reject request')
     def reject(self):
         assert self._reject_button is not None, 'Button not found'
         self._reject_button.click()
 
+    @allure.step('Open more options popup')
+    def open_more_options_popup(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOUT_MSEC, attempt: int = 2):
+        try:
+            self._open_canvas_button.click()
+            driver.waitFor(lambda: ContactsSettingsView()._view_profile_item.is_visible, timeout_msec)
+            return self
+        except:
+            if attempt:
+                self._open_canvas_button.click(attempt - 1)
+                return self
+            else:
+                raise
```

Comment:
not sure if we can be more specific on what to `raise` but if we can, please add something here (my suggestion is Lookup Error)